mqtt_bridge/util.py

```diff
@@ -12,8 +11,7 @@ def lookup_object(object_path: str, package: str='mqtt_bridge') -> Any:
     return obj
 
 
-extract_values = message_conversion.extract_values  # type: Callable[[rospy.Message], Dict]
-populate_instance = message_conversion.populate_instance  # type: Callable[[Dict, rospy.Message], rospy.Message]
+extract_values = message_conversion.extract_values  
```

Comment:
maybe you can use this for serialization, and you can drop `rosbridge_library` dependency.
- https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/serialization.py

Reply:
we don't have to include this in this PR.

Reply:
Was busy with some work lately, therefore could not reply, it is a nice idea to add this, but may be lets keep this for another PR.